tests/python/test_syntax_errors.py

```diff
@@ -274,20 +274,6 @@ def func():
         func()
 
 
-@test_utils.test()
-def test_redefining_template_args():
-    @ti.kernel
-    def foo(a: ti.template()):
```

Comment:
why is this test deleted btw? maybe good to keep one test with a as ti.template so that we know the expected behavior.

Reply:
emmm the error message changed... I'll add it back and change the error message.